### DISCUSSION TITLE
Permission to disable entries

### DISCUSSION THREAD

**Discussion by darylknight** [craftcms/cms#13734] [Ideas]
We're about to launch a site with several users groups. For context, we're also utilising the Workflow plugin to allow an Editor group to submit changes to a Publisher group for approval. Below is a screenshot of the permissions of the Editor group (these are the same for all sections)

![image](https://github.com/craftcms/cms/assets/1250017/72f4b009-6eb7-4e49-98dd-f77807d4ba23)

Today, the client asked if the Editor group could be allowed to disable entries.

Thinking this through, to disable an entry, you have to save it. Editors are not allowed to save entries because they require approval first. Is there a possibility to create a permission to disable entries, similar to how there is a permission for deleting enties?

**Comment by brandonkelly** [comment]
All permissions are designed to be mutually exclusive, so if we added this, it would require “Save entries” to no longer cover disabling entries, which would be the only exception to that permission.

That said, it might make sense for us to make it possible alongside some other governance/workflow features we’re starting to think about.

**Comment by darylknight** [comment]
Got it, thanks for the input :)
